src/views/Home/components/Banners/IFOBanner.tsx

```diff
@@ -1,7 +1,16 @@
+import { memo } from 'react'
 import styled from 'styled-components'
 import { Text, Flex, Button, ArrowForwardIcon, Heading } from '@pancakeswap/uikit'
 import { NextLinkFromReactRouter } from 'components/NextLink'
 import { useTranslation } from 'contexts/Localization'
+import useSWRImmutable from 'swr/immutable'
+import ifoV3Abi from 'config/abi/ifoV3.json'
```

Comment:
remove if no used. `Eslint` should warn this

Reply:
Done